server/src/test/java/org/elasticsearch/health/node/tracker/RepositoriesHealthTrackerTests.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.health.node.tracker;
+
+import org.elasticsearch.cluster.metadata.RepositoryMetadata;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.health.node.RepositoriesHealthInfo;
+import org.elasticsearch.health.node.UpdateHealthInfoCacheAction;
+import org.elasticsearch.repositories.InvalidRepository;
+import org.elasticsearch.repositories.RepositoriesService;
+import org.elasticsearch.repositories.Repository;
+import org.elasticsearch.repositories.RepositoryException;
+import org.elasticsearch.repositories.UnknownTypeRepository;
+import org.elasticsearch.test.ESTestCase;
+import org.junit.Before;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class RepositoriesHealthTrackerTests extends ESTestCase {
+
+    private RepositoriesHealthTracker repositoriesHealthTracker;
+
+    private RepositoriesService repositoriesService;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+
+        repositoriesService = mock(RepositoriesService.class);
+
+        repositoriesHealthTracker = new RepositoriesHealthTracker(repositoriesService);
+    }
+
+    public void test_getHealth_noRepos() {
+        when(repositoriesService.getRepositories()).thenReturn(Map.of());
+
+        var health = repositoriesHealthTracker.checkCurrentHealth();
+
+        assertTrue(health.unknownRepositories().isEmpty());
+        assertTrue(health.invalidRepositories().isEmpty());
+    }
+
+    public void test_getHealth_correctRepo() {
```

Comment:
Did you read somewhere in the guides about this test naming convention? As far as I know, we use camel case.

Reply:
This is the naming convention that I'm used to (and prefer), and I saw some other classes doing this as well (admittedly, only a couple), so I thought it was OK to choose for myself (like with `assertThat` and `assertEquals`). I can change it to be camel case if you think that's more in line with the project conventions.

Reply:
I understand this, I checked and you are right that there are tests using this convention but I think there are so few that I find hard to justify keeping this naming. While `assertEquals` and `assertThat` are more evenly spread. Does this make sense?